9.2/Dockerfile.rhel7

```diff
@@ -46,6 +43,9 @@ RUN yum install -y yum-utils gettext && \
     test "$(id postgres)" = "uid=26(postgres) gid=26(postgres) groups=26(postgres)" && \
     chmod -R go+rwx /var/lib/pgsql && chmod go+w /var/run/postgresql
 
+COPY run-*.sh /usr/local/bin/
+COPY contrib /var/lib/pgsql/
```

Comment:
@bparees I think it is worth to move the RUN after we add the script. Also add comment line about why we are making it world writeable.
I think @rhcarvalho patch is correct, he just needs to add that chmod after RUN.

I don't think that adding extra ~1m (if we do that for all images) to a build job is something that will hurt. The benefit is to have much friendlier developer workflow ;-)